src/components/Layout/Fixed.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Sidebar from '../Sidebar'
+import Timeline from '../Timeline'
+import propTypes from './propTypes'
+
+const Layout = ({ isOpen, tracks, now, time, timebar, toggleTrackOpen, clickElement }) => (
```

Comment:
Better to call this something like `Default` if necessary as it isn't really "fixed" (in the CSS sense).

Also `src/components/Layout/Sticky.jsx` declares `class StickyLayout extends PureComponent` so this should probably be `class DefaultLayout extends PureComponent` for consistency.